src/ActionBmwCodeCell.scss

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+.actionBmwCodeCell {
+  border: 2px solid blue;
+  background-color: gray;
+  border-radius: 5px;
+  padding: 1em;
```

Comment:
Can you change scss to use `rem` instead of `em`? The calculation of `em` is based on the size of its parent element while `rem` is always based on the single root element? Thanks.

This comment applies to elsewhere that `em` is used.

Reply:
Sure.  I didn't know about `rem` units.  It's good idea.